plugins.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE HTML>
+<!--
+	Arcana by HTML5 UP
+	html5up.net | @n33co
+	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
+-->
+<html>
+	<head>
+
+		<script async src="https://www.googletagmanager.com/gtag/js?id=UA-76450307-1"></script>
+		<script>
+			window.dataLayer = window.dataLayer || [];
+			function gtag(){dataLayer.push(arguments);}
+			gtag('js', new Date());
+
+			gtag('config', 'UA-76450307-1');
+		</script>
+
+		<title>Glue Plugins</title>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1" />
+		<link rel="icon" href="favicon.ico">
+		<!--[if lte IE 8]><script src="assets/js/ie/html5shiv.js"></script><![endif]-->
+		<link rel="stylesheet" href="assets/css/main.css" />
+		<!--[if lte IE 8]><link rel="stylesheet" href="assets/css/ie8.css" /><![endif]-->
+		<!--[if lte IE 9]><link rel="stylesheet" href="assets/css/ie9.css" /><![endif]-->
+	</head>
+	<body>
+		<div id="page-wrapper">
+
+			<!-- Header -->
+				<div id="header">
+
+					<!-- Logo -->
+						<h1><a href="index.html" id="logo">Glue:<em> multi-dimensional linked-data exploration</em></a></h1>
+
+					<!-- Nav -->
+						<nav id="nav">
+							<ul>
+								<li><a href="index.html">Home</a></li>
+								<li ><a href="install.html">Install</a></li>
+								<li><a href="http://docs.glueviz.org" target="_blank">Documentation</a></li>
+								<li><a href="team.html">Team</a></li>
+								<li><a href="get_involved.html">Get involved</a></li>
+								<li class="current"><a href="plugins.html">Plugins</a></li>
+							</ul>
+						</nav>
+
+				</div>
+
+			<!-- Main -->
+				<section class="wrapper style1">
+					<div class="container">
+						<div id="content">
+
+							<!-- Content -->
+
+								<article>
+									<header>
+										<h2>Glue Plugins</h2>
+									</header>
+
+<p>Glue’s standard data viewers (scatter plots, images, histograms) are useful in a wide variety of data exploration settings. However, they represent a tiny fraction of the ways to view a particular dataset. For this reason, Glue provides a way to create more deta viewers that me better suited to what you need.</p>

```

Comment:
The plugins aren't only for data viewers, but also e.g. data loaders and so on. I think here it would make sense to have a more general sentence here to say that glue is designed to make it easy to customize various aspects via plugins, including for example extra data viewers on top of the standard ones, readers/writers for new file formats, and more.

Reply:
I will edit the page to include the viewers as well and edit the paragraph above.